README.md

```diff
@@ -0,0 +1,91 @@
+# is-dom-node
+
+![Build Status](https://img.shields.io/travis/cjbarth/is-dom-node)
+![Code Coverage](https://img.shields.io/codecov/c/github/cjbarth/is-dom-node)
+![NPM Version](https://img.shields.io/npm/v/is-dom-node)
```

Comment:
This package name already exists and is a 6 year old package.
I guess we should not link to it and have to only add the badge once we are able to publish it with a scoped name?

(All other places using the package name in the readme and package.json need to be changes as well.)

Reply:
Your right. I went too quickly on this.